React-BC-2022.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
+  "folders": [
+    {
+      "path": "."
```

Comment:
What is the use of this?Not able to get.If u can explain it could be good.

Reply:
That's just my VS code setting, probably forgot to add this file to .gitignore; will take care in the next PR.